committing.rst

```diff
@@ -258,6 +258,35 @@ Developers can apply labels to GitHub pull requests).
 
 .. _cherry_picker.py: https://github.com/python/core-workflow/tree/master/cherry_picker
 
+.. _git_worktrees:
+
+Speeding up development workflow using git worktrees
+----------------------------------------------------
+It's often useful to have more than one checkout of the codebase (for testing on one version + 
+developing on another concurrently). For example::
+
+    # create a new clone of upstream/3.6 in ~/cpython_3.6 dir
+    $ git worktree add ~/3.6 upstream/3.6   
```

Comment:
Can you make the current working directory explicit?

Reply:
The '~' here and elsewhere presumes that one put the original clone in ones home directory, which I suspect is unusual.  Just use '../3.6' to put the worktree beside the clone, as one must be in the original clone for the git command to work.

I just checked, and '~/whatever' does not ever work on windows.  I am still adamantly opposed to making the devguide linux specific, including the $ crap.  We really did agree last spring to not do that.